src/Utils/processFulfillmentOptions.ts

```diff
@@ -0,0 +1,154 @@
+const EU_COUNTRY_CODES = [
+  "AT",
+  "BE",
+  "BG",
+  "HR",
+  "CY",
+  "CZ",
+  "DK",
+  "EE",
+  "FI",
+  "FR",
+  "DE",
+  "GR",
+  "HU",
+  "IE",
+  "IT",
+  "LV",
+  "LT",
+  "LU",
+  "MT",
+  "NL",
+  "PL",
+  "PT",
+  "RO",
+  "SK",
+  "SI",
+  "ES",
+  "SE",
+] as const
+
+type MPMoney = {
+  minor: any
+}
+
+type ShippingUnion = MPMoney | "ARTSY_SHIPPING" | null
+
+type FulfillmentOptions = {
+  artworkLocation: {
+    countryCode: string
+  }
+  pickupAvailable: boolean
+  onlyShipsDomestically: boolean
+  domesticShipping: ShippingUnion
+  internationalShipping: ShippingUnion
+  freeGlobalShipping: boolean
+  freeShippingToUserCountry?: boolean
+}
+
+interface ArtworkFulfillmentData {
+  readonly artsyShippingDomestic: boolean | null | undefined
+  readonly artsyShippingInternational: boolean | null | undefined
+  readonly shippingCountry: string | null | undefined
+  readonly domesticShippingFee:
+    | {
+        readonly minor: any
+      }
+    | null
+    | undefined
+  readonly euShippingOrigin: boolean | null | undefined
+  readonly internationalShippingFee:
+    | {
+        readonly minor: any
+      }
+    | null
+    | undefined
+  readonly isPurchasable: boolean | null | undefined
+  readonly onlyShipsDomestically: boolean | null | undefined
+  readonly pickupAvailable: boolean | null | undefined
+}
```

Comment:
responded to feedback by removing this which was previously a hook with a fragment. I just copied the generated type in but if we actually merged this (I think we shouldn't, this logic should go up to MP/Gravity) we might want to restrict the types in some other way to make sure the user queries the full data necessary (since everything is nullable)

Reply:
I know its hackathon and hacks are fine, but using the hook is probably better than copying out types from a generated fragment. (That feedback was just encouragement to try to obtain that relay data via the component fragment, vs a hook that's spread.)